Changelog.md

```diff
@@ -7,6 +7,9 @@ Language Features:
 
 Compiler Features:
  * Code Generator: Do not introduce new source references for small internal routines.
+ * Code Generator: Support calling bound library functions.
```

Comment:
We do not provide changelog entries for the Sol->Yul code generator while it is still experimental.

Reply:
OK. So I was right not to include it after all :)

What about #9074? Should tiny bugfixes like this be listed?

Reply:
Ewasm is also experimental, but I think it does not hurt to include it.

Reply:
OK. Generator lines removed, I left only the one for #9074.